utilities/backupable/backupable_db_test.cc

```diff
@@ -3568,6 +3568,90 @@ TEST_F(BackupEngineTest, BackgroundThreadCpuPriority) {
   DestroyDB(dbname_, options_);
 }
 
+// Populates `*total_size` with the size of all files under `backup_dir`.
+// We don't go through `BackupEngine` currently because it's hard to figure out
+// both the metadata file size and the size attributable to an incremental
+// backup.
```

Comment:
Btw, with BackupFileInfo, you can determine sizes with overlap, by de-duping the data yourself. But it does not include meta size.